.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+## Description
+
+*Replace this paragraph with a description of what this PR is doing. If you're
+modifying existing behavior, describe the existing behavior, how this PR is
+changing it, and what motivated the change. If you're changing visual
+properties, consider including before/after screenshots (and runnable code
+snippets to reproduce them).*
+
+## Related issues Issues
```

Comment:
Should the word issues be repeated here?

Reply:
Good catch. Thanks!